### ISSUE TITLE
This is abandoned 

### ISSUE THREAD

**Issue by alexa-v2** [vysheng/tg#1606]
Don't bother using it it's broken

**Comment by awsms** [comment]
it's running perfectly fine

**Comment by alexa-v2** [comment]
If you say so
Literally randomly can't send messages says peer invalid
Randomly disconnected
Hasn't been updated in forever


(Using Ubuntu 18.04)

**Comment by machetazo** [comment]
install gentoo

**Comment by awsms** [comment]
> install gentoo

based

**Comment by darkicekeel** [comment]
so what about telegram-cli on gentoo amd64 right now?

**Comment by Itookmyprozac** [comment]
Yes, it looks abandoned and it lacks documentation or instructions on how to use it.

Yes, it still works.

(Using Ubuntu 18.04).

**Comment by darkicekeel** [comment]
> Yes, it looks abandoned and it lacks documentation or instructions on how to use it.
> 
> Yes, it still works.
> 
> (Using Ubuntu 18.04).

what about gentoo? is it possible to run on it?
